packages/app/src/containers/Inspector.tsx

```diff
@@ -51,6 +52,10 @@ export default memo(function Inspector() {
             rotation: undefined,
           };
 
+    const hasTextLayer =
+      selectedLayers.length > 0 &&
```

Comment:
The `.some` should handle the empty array case correctly already, so we shouldn't need `selectedLayers.length > 0`